src/infra/runtimeQueryRepo.go

```diff
@@ -15,6 +15,23 @@ import (
 type RuntimeQueryRepo struct {
 }
 
+func (r RuntimeQueryRepo) GetVirtualHostPhpConfFilePath(
+	hostname valueObject.Fqdn,
+) (string, error) {
+	mainVirtualHost, err := infraHelper.GetPrimaryHostname()
+	if err != nil {
+		return "", errors.New("PrimaryHostnameNotFound")
+	}
+
+	primaryConfFile := "/app/conf/php/primary.conf"
```

Comment:
Se o vhost não for o primary, pq tá sendo usado o termo primary aqui?

Reply:
Está errado. Alterei o nome correto que é `phpConfFile`.